lib/widget/watcher/window_watcher.dart

```diff
@@ -52,23 +53,39 @@ class _WindowWatcherState extends ConsumerState<WindowWatcher> with WindowListen
     super.dispose();
   }
 
+  //Linux alternative for onWindowMoved and onWindowResized
+  @override
+  Future<void> onWindowMove() async {
+    if(checkPlatform([TargetPlatform.linux]) && !timeout) {
+      timeout = true;
+      final windowOffset = await windowManager.getPosition();
+      final windowSize = await windowManager.getSize();
+      await _dimensionsController?.storeDimensions(windowOffset: windowOffset, windowSize: windowSize);
+      Timer(const Duration(milliseconds: 600), () => toggleTimeout());
```

Comment:
I think it is better to store the end timestamp instead of a boolean so you don't need to create a timer.

Reply:
Used a Stopwatch for overall better syntax with elapsedtime.